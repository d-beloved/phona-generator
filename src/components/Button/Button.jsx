import React from 'react';

const Button = ({type, disabled, onClick, text, className}) => (
  <button
    type={type}
    disabled={disabled}
    onClick={onClick}
    className={className}
  >
    {text}
  </button>
);

export default Button;
