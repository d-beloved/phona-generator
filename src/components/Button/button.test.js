import React from 'react';
import { shallow } from 'enzyme';

import Button from "./Button";

describe('Button Component', () => {
  it('should render the button correctly', () => {
    const component = shallow(<Button />);

    expect(component).toMatchSnapshot();
  });
});
