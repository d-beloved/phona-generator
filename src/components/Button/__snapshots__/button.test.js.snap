// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component should render the button correctly 1`] = `<button />`;
